src/porepy/utils/graph.py

```diff
@@ -65,6 +61,8 @@ def bfs(self, start, color):
             node = queue.pop(0)
             if node not in visited:
                 visited.append(node)
-                neighbours = sparse_mat.slice_indices(self.node_connections, node)
+                neighbours = pp.matrix_operations.slice_indices(
```

Comment:
Consider changing to neighbors to be consistent with rest of PorePy (US grammar). Mixing has lead to cumbersome debugging in the past!

Reply:
Fair point. Will do.